lib/Server.js

```diff
@@ -569,8 +563,8 @@ class Server {
         this.sockWrite([connection], 'liveReload');
       }
 
-      if (this.options.progress) {
-        this.sockWrite([connection], 'progress', this.options.progress);
+      if (this.options.client.progress) {
+        this.sockWrite([connection], 'progress', this.options.client.progress);
       }
 
       if (this.options.clientOverlay) {
```

Comment:
I just realized this should be `this.options.client.overlay`.

Reply:
yes, let's fix it in the separate PR